src/NuGet.Core/NuGet.Test.Utility/SimpleTestSetup/SimpleTestPackageUtility.cs

```diff
@@ -38,14 +39,17 @@ public static class SimpleTestPackageUtility
            string repositoryDir,
            string id,
            string version,
-           IEnumerable<PackageDependency> dependencies)
+           IEnumerable<PackageDependency> dependencies,
+           bool isSymbolPackage = false)
```

Comment:
Add a new constructor instead of an optional parameter, as a generic type like this it gets painful